src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -1343,6 +1346,7 @@ def cli(self, interface='', output=None):
                 if 'encapsulation' not in ret_dict[intf]:
                     ret_dict[intf]['encapsulation'] = {}
                 ret_dict[intf]['encapsulation']['native_vlan'] = m.groupdict()['native_vlan']
+                ret_dict[intf]['encapsulation']['native_vlan_name'] = m.groupdict()['native_vlan_name']
```

Comment:
and the sam thing here about the key being `dummy` and not `native_vlan_name`
https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxe/show_interface.py#L1340

Reply:
Must have been missed in my commit. Will get corrected.

Reply:
great! saw that the sample output line is already in

Reply:
Should be corrected now. Apologies about that.

Reply:
Np, looks good now, thanks!